### PR TITLE
Ro'zimuhammad changed code

### PR DESCRIPTION
**Pull request by Akhmadovich03**

### CODE REVIEW

File: Lesson01/LMS/MainWindow.xaml.cs [hunk 1]
```diff
@@ -23,18 +23,20 @@ public partial class MainWindow : Window
...
         {
             var window = new StackWindow();
+
+            window.Show();
         }
 
         private void Button_Click_1(object sender, RoutedEventArgs e)
         {
             var window = new GridWindow();
+            window.Show();
```
Comment (Mirazyzz) [inline Lesson01/LMS/MainWindow.xaml.cs:39]: no empty line.

**Review by Mirazyzz — CHANGES_REQUESTED** [review]

**Review by Boburm1rzo — COMMENTED** [review]
kamchiliklar bor

**Comment by Mirazyzz** [comment]
Please, resolve conflicts before merging.

**Review by Akhmadovich03 — COMMENTED** [review]
changed file

**Review by Mirazyzz — APPROVED** [review]
LGTM!

File: Lesson01/LMS/MainWindow.xaml.cs [hunk 2]
```diff
@@ -10,20 +10,20 @@ public partial class MainWindow : Window
         public MainWindow()
         {
             InitializeComponent();
-
-            string a = "hello";
-            int num = 3;
+            //Ro'zimuhammad's change
```
Comment (Mirazyzz) [inline Lesson01/LMS/MainWindow.xaml.cs:13]: Remove the comment

**Review by Mirazyzz — CHANGES_REQUESTED** [review]

**Review by Boburm1rzo — APPROVED** [review]
nice

**Review by Akhmadovich03 — COMMENTED** [review]
cleared comment

**Review by Mirazyzz — APPROVED** [review]
Looks good!

**Pull request merged**